### PR TITLE
Fixed commit version downloads

### PR DESCRIPTION
**Pull request by deitch**
In 3 places in building various `pkg/*`, in the Dockerfile, we downloaded from the Internet or used the `busybox` image with just the latest. This meant that reproducing it was not reliable, as `latest` for an OCI image or a download link could change.

3 commits in this PR each addresses it for 3 different `pkg/*/Dockerfile`:

* `pkg/strongswan`: explicitly specify a version rather than just downloading link to latest. Note that I checked which version we currently are using via the md5 hash and used the same one. This PR **does not** change the version used, only explicitly references it
* `pkg/fw`: use an explicit `FROM busybox@sha256:<hash>` instead of just `FROM busybox`
* `pkg/uefi`: check out the specific commits of edk2 and opensbi instead of just cloning and using whatever the most recent commit is

### CODE REVIEW

**Comment by deitch** [comment]
The builds are failing due to the issue fixed in #2773 . On hold until that merges in, then I can rebase.

**Comment by deitch** [comment]
yetus should be fine now.

**Comment by deitch** [comment]
Rebased now that #2773 is in

**Review by zededa-yuri — APPROVED** [review]

**Comment by deitch** [comment]
I do not understand the test failures. We changed nothing, just pinned the existing versions of dependencies we were using.

File: pkg/fw/Dockerfile
Comment (giggsoff) [inline pkg/fw/Dockerfile:48]: Probably it would be better to use `FROM ${EVE_BUILDER_IMAGE} as compactor` to not use additional external entities here.

Comment (deitch) [reply pkg/fw/Dockerfile:48]: I am fine with that; I was trying not to change anything.

Now that I look at it, it doesn't do anything in `compactor`. It just copies over the various `/lib/firmware/` from the `build` image, and then runs `cp` to create a symlink. Is that it? The entire purpose is for the symlink? If so, then, yes, let's adopt your approach. Doing it now.

Comment (deitch) [reply pkg/fw/Dockerfile]: Done.

**Comment by deitch** [comment]
Much better. Now it looks like it passed. @eriknordmark do you want to kick off the eden tests?

**Review by eriknordmark — APPROVED** [review]
Run eden

**Comment by deitch** [comment]
Again eden tests are failing. I am not familiar enough with the specific tests to know if these are actually failures or just transient issues?

**Comment by deitch** [comment]
Eden tests still failing. Interestingly most pass. Only `tpm=false;zfs` fails; even `tpm=true;zfs` passes. 🤷‍♂️

**Pull request merged**
